apps/account/tests/test_views/test_user_change_password_view.py

```
@@ -1,47 +1,39 @@
 
-        self.assertIn("Change your password", str(response.content))
+    assert response.status_code == http.HTTPStatus.OK
+    assert response.template_name[0] == UserChangePasswordView.template_name
+    assert "Change your password" in response.content.decode()
 
-    def test_post_regular(self):
-        new_password = "my_new_password"
 
-        client = self.reauthenticate_user(self.user)
+def test_post_regular(authenticated_client, user):
+    new_password = "my_new_password"
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "new_password"

(S105)